src/quo2/components/inputs/locked_input/component_spec.cljs

```diff
@@ -0,0 +1,16 @@
+(ns quo2.components.inputs.locked-input.component-spec
+  (:require [quo2.components.inputs.locked-input.view :as locked-input]
+            [test-helpers.component :as h]))
+
+(h/describe "Locked Input"
+            (h/test "renders label, value and icon"
+                    (h/render [locked-input/locked-input
+                               {:label-text "Label"
+                                :value-text "Value"
+                                :icon       "🔫"}])
+                    (-> (js/expect (h/get-by-text "Label"))
```

Comment:
there is a helper `h/is-truthy` it's better if you can use that here instead :)

Reply:
Personally I would add some more tests, with various permutations of these. eg label no value, etc 👍

Reply:
Done